FirebaseMLModelDownloader/Sources/ModelInfoRetriever.swift

```diff
@@ -194,17 +179,12 @@ extension ModelInfoRetriever {
 
   /// Save model info to user defaults.
   func saveModelInfo(data: Data, modelHash: String) {
-    // TODO: Save model info to user defaults
+    let decoder = JSONDecoder()
+    guard let modelInfoJSON = try? decoder.decode(ModelInfoResponse.self, from: data)
+    else { return }
+    modelInfo = ModelInfo(app: app, name: modelName)
```

Comment:
nit: I would create a local non-option variable to avoid extra optional chains `?` below:
```
let modelInfo = ModelInfo(app: app, name: modelName)
...
self.modelInfo = modelInfo
```

Reply:
Done.